docs/TRIBE.md

```diff
@@ -60,6 +60,9 @@ Name 	    Number of Members 	 plugins 	 tasks
 all-nodes 	1 			         0   		 0
 ```
 
+#### Quick example for creating and joining agreements:
+![tribe-create-join-agreement](https://user-images.githubusercontent.com/18285077/28823052-0ea6b9ca-76bc-11e7-97d9-16b9b0018709.gif)
```

Comment:
Starting a 4 node cluster and listing members should be presented as first thing - see how it was done in the previous version of [TRIBE.md](https://github.com/intelsdi-x/snap/blob/fc2219a5f30fd1b32cc16379f21b109e69e3c73b/docs/TRIBE.md)

Example of creating an agreement and joining members should be shown as the next thing